src/schema/create-index-builder.ts

```diff
@@ -138,6 +138,43 @@ export class CreateIndexBuilder<C = never>
     })
   }
 
+  /**
+   * Add INCLUDE clause with columns to the index.
+   *
+   * Also see {@link expression} for
+   * specifying an arbitrary expression.
+   *
+   * ### Examples
+   *
+   * ```ts
+   * await db.schema
+   *         .createIndex('person_first_name_include_age_index')
+   *         .column('first_name')
+   *         .include(['age'])
+   *         .on('person')
+   *         .execute()
+   * ```
+   *
+   * The generated SQL (PostgreSQL):
+   *
+   * ```sql
+   * create index "person_first_name_include_age_index" on "person" ("first_name") INCLUDE ("age")
+   * ```
+   */
+    include<CL extends string>(
+      columns: OrderedColumnName<CL>[],
```

Comment:
Why `OrderedColumnName<CL>` and not just `string` without any generics? There's no test that uses the ordered versions at least. If those are supported, add tests for them.